docs/monitoring/flow.rst

```diff
@@ -0,0 +1,68 @@
+.. |MonitorNorthRate| image:: ../imagesMonitorNorthRate.jpg
+.. |MonitorZendesk| image:: ../imagesMonitorZendesk.jpg
+.. |MonitorTrigger| image:: ../images/MonitorTrigger.jpg
+.. |MonitorBuffered| image:: ../images/MonitorBuffered.jpg
+
+Pipeline Data Flow
+==================
+
+There are several aspect to monitoring the data pipelines within Fledge, the one we will discuss in this section is how to monitor the flow of data. This is about if data is flowing, not if the data that is flowing is good data, just because there is a flow of data says nothing about the quality of that data.
+
+The south services automatically monitor the flow rates out from the south service to the storage layer, the configuration of this monitoring is discussed in the section on tuning Fledge. This automatic monitoring uses statistical methods to detect changes in the flow rate and alerts the user via the user interface status bar alerts.
+
+The main mechanism for monitoring data flow is the statistics that are collected by all the services that implement a Fledge instance.
+
+South Service Data Flow
+-----------------------
+
+The south service maintains two distinct statistics, one for the total ingest count for the services and a set for the ingest counts for each asset ingested by the service. These statistics are collected when the data is sent from the south service to the storage service, they do therefore not include any readings that have been bought into the service by removed by any of the filters in the filter pipeline.
```

Comment:
- "...two distinct statistics, one for the total ingest..." Change comma to a colon.
- "...south service to the storage service, they do therefore..." Change comma to a sem-colon.
- "...been bought into the service by removed by any of the filters..." I think this should be "...or removed by any of the filters..."